extension/entitystore/eksInfo.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package entitystore
+
+import (
+	"time"
+
+	"github.com/jellydator/ttlcache/v3"
+	"go.uber.org/zap"
+)
+
+const ttlDuration = 5 * time.Minute
```

Comment:
How did we decide on 5 minutes?

Reply:
Copied from original PR:
> As discussed on slack, ttl cache will be 5 minutes. I misunderstood, since most customers publish minutely, 5 minutes should be a good indicator of whether a pod is still active. Even if a pod passes a metric after its been terminated, we are okay since the cache will just reconstruct

Reply:
Non-blocking comment